cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,33 @@
 /// <reference types='cypress' />
 
+const firstName = 'Anderson';
+const lastName = 'Konstant';
+const userEmail = 'Anderson@gmail.com';
+const phonnumber = 802654594154;
```

Comment:
The variable `phonnumber` should be renamed to `phoneNumber` to follow camelCase naming conventions.